### PR TITLE
 Pass --no-color to lldb instead of using a dumb terminal to disable color

### PR DESCRIPTION
**Pull request by lanza**

### CODE REVIEW

File: lib/lldbwrap.sh [hunk 1]
```diff
@@ -42,6 +42,4 @@ cleanup()
 trap cleanup EXIT
 
 # Execute lldb finally with our custom initialization script
-# Use dumb terminal to suppress prompt coloring, which is filtered by
-# neovim anyway.
-TERM=dumb "$lldb" -S $lldb_init $rest
+"$lldb" -S $lldb_init $rest --no-use-colors
```
Comment (sakhnik) [inline lib/lldbwrap.sh]: Could you please move the option `--no-use-colors` ahead? Otherwise, it could fail if `$rest` contained `--`.

**Comment by sakhnik** [comment]
That's an awesome suggestion. If there was a way to supply a custom marker symbols to the prompt like `gdb.prompt_hook` in GDB, we could well preserve colors.

**Comment by lanza** [comment]
> That's an awesome suggestion. If there was a way to supply a custom marker symbols to the prompt like gdb.prompt_hook in GDB, we could well preserve colors.

I still intend to replace the scraping of the text with a more integrated solution (e.g. in the outstanding pull request I have up) that will have lldb telling nvim-gdb when breakpoints are modified and when stops/starts happen. I just haven't been debugging native code much in the past few months so I haven't gotten around to it. 

Once that is in colors can easily be restored.

**Comment by sakhnik** [comment]
Well, the change is good now. Thanks!

**Pull request merged**